en/dev_log/logging.md

```diff
@@ -100,33 +100,27 @@ amount of dropouts:
 
 ## SD Cards
 
-The following provides performance results for different SD cards.
-Tests were done on a Pixracer; the results are applicable to Pixhawk as well.
-
-:::note
 The maximum supported SD card size for NuttX is 32GB (SD Memory Card Specifications Version 2.0).
```

Comment:
I'm not sure if this one still applies

Reply:
@davids5 Can you comment on the current SD spec supported by PX4/NuttX and what this means for the maximum card size?

Reply:
> @davids5 Can you comment on the current SD spec supported by PX4/NuttX and what this means for the maximum card size?

I believe it is 32GB. 

I feel strongly you should not remove that table, But add 4 columns and extend the header  to say |F4 based Flight Controllers| |F7 based Flight Controllers| |H7 based Flight Controllers| - The information is valuable

Then get the some number from the discussion on discord and request some testing to fil it out.